packages/genesys-spark-components/src/components/stable/gux-listbox/gux-listbox.service.ts

```diff
@@ -216,17 +217,18 @@ export function matchOption(
   option: ListboxOptionElement,
   matchString: string
 ): boolean {
-  //The text content needs to be trimmed as white space can occur around the textContent if options are populated asynchronously.
-  return getOptionDefaultSlot(option)
-    ?.textContent.trim()
+  return getOptionDefaultSlotText(option)
     .toLowerCase()
     .startsWith(matchString.toLowerCase());
 }
 
-export function getOptionDefaultSlot(
-  option: ListboxOptionElement
-): Node | undefined {
-  return option.shadowRoot.querySelector('slot')?.assignedNodes()[0];
```

Comment:
I opted to completely replace the prior method with the new one that returns the text content as a string. All the prior usages were doing the same `.textContent.trim()` thing anyway, and its main problem was that it returned only the first node from `assignedNodes()`. I'm open to keeping it if this would be a breaking change or if there are use cases I'm not thinking of.